src/main/scala/za/co/absa/abris/examples/using_keys/KafkaAvroWriterWithKey.scala

```diff
@@ -46,35 +46,22 @@ object KafkaAvroWriterWithKey {
   def main(args: Array[String]): Unit = {
 
     // there is a sample properties file at /src/test/resources/DataframeWritingExample.properties
-    if (args.length != 1) {
-      println("No properties file specified.")
-      System.exit(1)
-    }
-
-    println("Loading properties from: " + args(0))
-    val properties = loadProperties(args(0))
-
-    for (key <- properties.keysIterator) {
-      println(s"\t${key} = ${properties.getProperty(key)}")
-    }
+    checkArgs(args)
```

Comment:
I would be explicit that these are methods imported from `ExamplesUtils`, but that's a matter of personal preference.